.eslintrc.js

```diff
@@ -49,6 +50,8 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'brace-style': ["error", "stroustrup", { "allowSingleLine": true }],
+    // disable linebreak-style rule
+    'linebreak-style': 'off'
```

Comment:
please remove these ES lint rules changes

Reply:
The disabling of the es lint rules was was an issue from https://github.com/creativecommons/cccatalog-frontend/pull/654

Reply:
Yep, but it's not within the scope of this PR. Please open a separate PR with this change.

Reply:
I'll reopen this PR https://github.com/creativecommons/cccatalog-frontend/pull/654 then